lib/iris/tests/unit/util/test_new_axis.py

```diff
@@ -136,6 +139,17 @@ def test_maint_factory(self):
         self.assertEqual(res, com)
         self._assert_cube_notis(res, cube)
 
+        # Check that factory dependencies are actual coords within the cube.
+        # Addresses a former bug : see XXXX
```

Comment:
I assume you'll update this with the issue number?

Reply:
Ok